sensor_tag_read_block.py

```diff
@@ -147,63 +145,51 @@ def _scan_helper(self):
             self._configs[addr] = AttributeDict({
                 'address': addr,
                 'name': "{}-{}".format(self.default_metadata.name, len(self._configs)),
-                'read_interval': self.default_metadata.read_interval,
                 'sensors': self.default_metadata.sensors
             })
 
     def connect(self):
-        spawn(self._connect_tags)
-        
-    def _connect_tags(self):
+        connecting_to = []
         for addr in self._configs:
             if addr not in self._tags:
-                self._connect_tag(self._configs[addr])
-            if addr not in self._sensors:
-                sensors = self._get_sensors(*self._tags[addr])
-                [s.enable() for s in sensors]
-                self._sensors[addr] = sensors
-        self._schedule_read_jobs()
-
-    def _connect_tag(self, cfg):
+                connecting_to.append(addr)
+                spawn(self._connect_tag, self._configs[addr])
+        return connecting_to
```

Comment:
this returns connecting_to but this is not used in `start` (line 97).

Is this for a later feature?